### PR TITLE
Update 13_Loading_Admin_UI_Assets.md

### PR DESCRIPTION
**Pull request by jdreesen**
Use `PathsEvent::addPaths()` instead of `PathsEvent::setPaths(array_merge(PathsEvent::getPath(), [...]))`

### CODE REVIEW

File: doc/20_Extending_Pimcore/13_Bundle_Developers_Guide/13_Loading_Admin_UI_Assets.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ class AdminAssetsListener implements EventSubscriberInterface
 
     public function onJsPaths(PathsEvent $event): void
     {
-        $event->setPaths(array_merge($event->getPaths(), [
+        $event->addPaths([
             '/bundles/app/js/admin.js'
         ]));
```
Comment (NiklasBr) [inline doc/20_Extending_Pimcore/13_Bundle_Developers_Guide/13_Loading_Admin_UI_Assets.md]:
```suggestion
        ]);
```

Comment (jdreesen) [reply doc/20_Extending_Pimcore/13_Bundle_Developers_Guide/13_Loading_Admin_UI_Assets.md]: :woman_facepalming: thx!

**Comment by blankse** [comment]
@jdreesen Can you change it also here?
https://github.com/pimcore/pimcore/blob/11.x/bundles/TinymceBundle/README.md

There is also the return type hint `: void` missing.

**Comment by jdreesen** [comment]
Sure, done.

---

I would like to take this opportunity to point out that I will be on holiday from the end of the week until the end of the month.

**Comment by blankse** [comment]
Thank you, have a great holiday.

**Comment by fashxp** [comment]
Thx and enjoy your time off

**Pull request merged**